requirements-freqai-rl.txt

```diff
@@ -3,8 +3,8 @@
 
 # Required for freqai-rl
 torch==1.13.1
-stable-baselines3==1.7.0
+stable-baselines3==1.7.0; sys_platform != "win32"
```

Comment:
I disagree with this.

CI clearly shows that both installation, as well as tests are running fine on windows.
It's therefore most likely related to your specific setup.